formatTest/unit_tests/expected_output/features403.re

```diff
@@ -13,7 +13,11 @@ type nonrec u 'a =
 type expr 'a =
   | Val {value: 'a} :expr 'a
   | Add
-      {left: expr int, right: expr int} :expr int
+      {
+        left: expr int,
+        right: expr int
+      }
+      :expr int
```

Comment:
A space after the `:` for the constructor's type may help in either case and is more consistent with the formatting used in records:

```
| Add {
    left: expr int,
    right: expr int
  } : expr int
```
```
| Add {
    left: expr int,
    right: expr int
  }
  : expr int
```
Looking at those two example, I like that the first is more compact.  In the second the `: expr int` looks a little strange on its own.  I think either would work.